lib/resolve.ts

```diff
@@ -104,3 +111,23 @@ export function resolveTask(
 
 	return task;
 }
+
+/**
+ * Given a balena architecture string, translate it to the equivalent
+ * docker platform string.
+ */
+function resolveDockerPlatform(balenaArchitecture: string): string | null {
+	switch (balenaArchitecture) {
+		case 'amd64':
+			return 'linux/amd64';
+		case 'i386':
```

Comment:
On the following page, I see an additional balena architecture: `i386-nlp`
https://www.balena.io/docs/reference/base-images/devicetypes/

Should it be mapped to `linux/386` as well? Maybe @nghiant2710 knows the answer, as I've found issue balena-io/docs/issues/1165 :-)

Reply:
I would love to have a definitive source on what platforms we support. probably should look through the supported device types to be sure